content/ja/docs/concepts/policy/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Policies"
```

Comment:
I expected to see Japanese text here. I don't know Japanese well enough to say whether the English word lacks a good equivalent.